packages/skott/src/skott.ts

```diff
@@ -485,12 +528,133 @@ export class Skott<T> {
     };
   }
 
+  private getValidGroup(nodePath: string) {
+    if (!this.config.groupBy) {
```

Comment:
This check does not seem relevant as we will never be in that case, given the `this.config.groupBy` is already checked in `makeProjectStructure` hence `getValidGroup` will be called only if `this.config.groupBy` is not undefined anyway

If it was used for type narrowing, at that point you can safely assume that `config.groupBy` is defined

Reply:
The idea was to throw an explicit error early, if this method will be used somewhere else in the future 🤔 

But, probably, normal `TypeError` like `this.config.groupBy is not a function` is also explicit enough + at the moment it is not possible to get to this exception anyway, so i will remove that ✅